### PR TITLE
trivial: replace Bugzilla comments with links

### PR DESCRIPTION
**Pull request by WalterBright**
1. replace Bugzilla numbers with Bugzilla links
2. replace C++ anachronisms with D equivalents
3. fix header comment to be Ddoc

### CODE REVIEW

File: src/dtemplate.d [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-// http://www.digitalmars.com
-// Distributed under the Boost Software License, Version 1.0.
-// http://www.boost.org/LICENSE_1_0.txt
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Template implementation.
+ *
+ * Copyright:   Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
```
Comment (wilzbach) [inline src/dtemplate.d]: If you already do this cleanup, may I ask a dumb question as I don't know much about the history: shouldn't we start to license as much as possible to the D Foundation?

**Comment by JackStouffer** [comment]
> replace Bugzilla numbers with Bugzilla links
replace C++ anachronisms with D equivalents
fix header comment to be Ddoc

If you do all of these things in different commits it makes it easier to review and to revert if something goes wrong

**Comment by WalterBright** [comment]
> If you do all of these things in different commits it makes it easier to review, and easier to revert if something goes wrong

They're all comments, and none of them are interrelated. That's why it's marked "trivial", and you can trivially verify it for yourself.

File: src/dtemplate.d (hunk 1 above)
Comment (WalterBright) [inline src/dtemplate.d]: one thing at a time

**Comment by JackStouffer** [comment]
I would have a hard time claiming that a 100+ line change is trivial :/

**Comment by andralex** [comment]
Auto-merge toggled on

**Pull request merged**